### DISCUSSION TITLE
Adding print classes, can't override display:block

### DISCUSSION THREAD

**Discussion by jacargentina** [tailwindlabs/tailwindcss#8372] [General]
```jsx
<div id="desktop-view" className="hidden md:flex overflow-hidden" />
```

But for printing I need `desktop-view` to be `display: block` so:

```jsx
<div id="desktop-view" className="hidden md:flex print:block overflow-hidden" />
```

Then:

<img width="454" alt="Captura de Pantalla 2022-05-17 a la(s) 20 45 03" src="https://user-images.githubusercontent.com/517228/168929785-b8e01ba3-b769-411b-96f8-6ea767f997a4.png">

It seems `md:flex` wins there? Why?

How can I achieve it?

**Comment by adamwathan** [comment]
Try `md:print:block` — I'm guessing the `md:` variant is still matching because the print size is above that breakpoint and that the breakpoint variants are defined later in the CSS than the `print` variant.
